webrtc/videoroom/lib/videoroom/pipeline.ex

```diff
@@ -171,17 +205,25 @@ defmodule VideoRoom.Pipeline do
         end)
 
     spec = %ParentSpec{children: children, links: links}
-    state = update_in(state, [:tracks, track_id], &%Track{&1 | encoding: encoding})
+    endpoint = Endpoint.update_track_encoding(state.endpoints[endpoint_id], track_id, encoding)
+    state = put_in(state.endpoints[endpoint_id], endpoint)
     {{:ok, spec: spec}, state}
   end
 
-  def handle_notification({:signal, message}, {:endpoint, peer_pid}, _ctx, state) do
-    send(peer_pid, {:signal, message})
-    {:ok, state}
+  def handle_notification(
+        {:vad, val} = msg,
+        {:endpoint, endpoint_id} = from,
+        _ctx,
+        state
+      ) do
+    Membrane.Logger.info("#{inspect(msg)}, from: #{inspect(from)}")
```

Comment:
Isn't it rather a debug level information than info?

Reply:
Acctually this should be totally removed